README.md

```diff
@@ -162,6 +162,25 @@ httpPost {
 }
 ```
 
+You may also specify a custom content type in the request's header :
+```kotlin
+httpPost {
+    header { "Content-Type" to "text/html" }
```

Comment:
Not sure that we need to recommend this option.
It’s better to mention in body javadoc.